lib/push.js

```diff
@@ -19,16 +19,24 @@ module.exports = function (cfg) {
     headers: headers
   };
 
-  var sender = AeroGear.Sender(settings);
 
   // $fh.push
+  var sender;
   return function push(message, options, callback) {
+    if (!aeroGearLoaded) {
+      AeroGear = require("unifiedpush-node-sender");
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add the "var" keyword to this declaration of "AeroGear". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://metrics.skunkhenry.com/coding_rules#rule_key=javascript%3AS2703)